Catappult/appcoins-wallet-android
# [APPC-2278] Vouchers for unknown users

**What does this PR do?**

Adds the layout changes requested by design for unknown users with vouchers

**Database changed?**

   No

**Where should the reviewer start?**

PromotionsPresenter.kt 

**How should this be manually tested?**

Check if an unknown user with vouchers is able to see the vouchers as shown in the design

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/APPC-2278

**Code Review Checklist**
